Packages/MIES/MIES_DAEphys.ipf

```diff
@@ -2716,7 +2716,13 @@ static Function DAP_CheckHeadStage(device, headStage, mode)
 			needResetting = 1
 		endif
 
-		if((!CheckIfClose(DAGain, DAGainMCC, tol=1e-4) && clampMode != I_EQUAL_ZERO_MODE) || (clampMode == I_EQUAL_ZERO_MODE && !CheckIfSmall(DAGainMCC)))
+		if(clampMode == I_EQUAL_ZERO_MODE && !CheckIfSmall(DAGainMCC))
+			printf "(%s) The DA gain for the DA channel %d in I=0 clamp mode is expected to be zero but the configured gain is %g.\r", device, DACchannel, DAGainMCC
```

Comment:
@t-b In I=0, the MCC returns a DA gain of zero? It makes sense that the amp would have a gain of 0, but does it matter what the DAC DA gain is?

Reply:
I've just rewritten the existing check here. I can remove it if you prefer that. I think it also makes sense to remove it as were are in all other places checking that the MIES settings are good not that the MCC has the correct settings by itself.